backend/.env.example

```
@@ -96,14 +98,19 @@ WEBHOOK_HEADERS=
-# IMPORTANT: These keys are used for encrypting sensitive data including git tokens
+# AES encryption configuration for share tokens and generic sensitive data
+# IMPORTANT: GIT_TOKEN_AES_* is used by shared.utils.crypto.encrypt_sensitive_data,
+# including Connector App fixed provider headers stored in kinds.
 # SHARE_TOKEN_AES_KEY must be exactly 32 bytes (32 characters) for AES-256
 # SHARE_TOKEN_AES_IV must be exactly 16 bytes (16 characters) for AES IV
+# GIT_TOKEN_AES_KEY must be exactly 32 bytes (32 characters) for AES-256
+# GIT_TOKEN_AES_IV must be exactly 16 bytes (16 characters) for AES IV
 # SECURITY WARNING: Change these default values in production!
 # Generate secure random keys using: openssl rand -hex 32 (for key) and openssl rand -hex 16 (for IV)
 SHARE_TOKEN_AES_KEY=12345678901234567890123456789012
 SHARE_TOKEN_AES_IV=1234567890123456
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SHARE_TOKEN_AES_IV key should go before the SHARE_TOKEN_AES_KEY key

(UnorderedKey)